packages/vats/src/localchain.js

```diff
@@ -91,6 +95,14 @@ const prepareLocalChainAccount = zone =>
         const system = getPower(powers, 'system');
         return E(system).toBridge(obj);
       },
+      async interceptTransfers(tap) {
```

Comment:
this behavior is surprising. if you call `interceptTransfers(tap)` is starts intercepting. but if you call `interceptTransfers()` it does the opposite (stops).

What's the rationale? Why not separate methods?